### PR TITLE
ODM-11970 GET files metadata endpoint

### PR DESCRIPTION
**Pull request by natalyatodorova**

### CODE REVIEW

File: openapi/v1/aFileUser.yaml [hunk 1]
```diff
@@ -109,7 +109,118 @@ paths:
...
+        Single words can be supplied as is, otherwise use speech marks (`"`) to quote queries that include whitespace. Speech marks and backslash characters in the query need to be escaped with a backslash (`\`).
+
+        ## Paging
+        For performance reasons, this endpoint returns results in "pages" of limited size. In order to retrieve all matching results the client needs to request multiple pages starting from an offset to the first result. You can do this using the `pageOffset` query parameter. A value of 0 (the default) instructs the server to return the first page of results, 100 would return a page of results starting from the 101st result and so on. To retrieve all results, iterate through pages by increasing the offset in multiples of limit (e.g., `offset = n * limit`), until all results have been retrieved. The total number of pages can be calculated by dividing the total number of results by the limit.
+
+        ## List operation
+
+        This endpoint can be called without any query parameters. When called this way, it returns a list of all files metadata objects.
+      parameters:
+        - in: path
```
Comment (m-smazhevsky) [inline openapi/v1/aFileUser.yaml]: I thought it should be `in: query`

**Review by m-smazhevsky — CHANGES_REQUESTED** [review]
please provide swagger screenshots

**Comment by natalyatodorova** [comment]
> please provide swagger screenshots

The sections are too long. Better to check right away on my instance https://develop-nt.dev.gs.team/swagger/?urls.primaryName=aFileUser

Below are screenshots w/o errors:
image
Screenshot 2024-11-14 at 09 54 17
Screenshot 2024-11-14 at 09 54 31
image

File: openapi/v1/aFileUser.yaml (hunk 1 above)
Comment (natalyatodorova) [reply openapi/v1/aFileUser.yaml]: Fixed. Thanks for noticing

**Review by m-smazhevsky — APPROVED** [review]

**Pull request merged**